mri/scripts/gridsearch.py

```diff
@@ -262,6 +262,7 @@ def launch_grid(kspace_data, reconstructor_class, reconstructor_kwargs,
                     'samples': fourier_op.samples,
                     'shape': fourier_op.shape,
                     'n_coils': fourier_op.n_coils,
+                    'implementation': 'gpuNUFFT', 
```

Comment:
can you add a description of this change in the PR description?

Reply:
Oh noooo, this was not supposed to be here, i assumed this PR is closed.. This is another issue. I will get another PR for this..